src/worker/tasks/manageChainIndexers.ts

```diff
@@ -0,0 +1,23 @@
+import { getContractSubscriptionsUniqueChainIds } from "../../db/contractSubscriptions/getContractSubscriptions";
+import {
+  INDEXER_REGISTRY,
+  addChainIndexer,
+  removeChainIndexer,
+} from "../indexers/chainIndexerRegistry";
+
+export const manageChainIndexers = async () => {
```

Comment:
this step here:

- it's creating cron-jobs and then clearing the created cron-jobs??

Reply:
yup. If indexer no longer needed, we take down the cron job. No reason to run a job if chain is no longer indexed